reyna/src/com/b2msolutions/reyna/Dispatcher.java

```diff
@@ -108,8 +119,12 @@ public static Result canSend(Context context) {
         return Result.OK;
     }
 
-    private static boolean canSendNow(BlackoutTime blackoutTime, String window) throws ParseException {
-        return blackoutTime.canSendAtTime(new GregorianCalendar(), window);
+    private static boolean canSendNow(BlackoutTime blackoutTime, String window, GregorianCalendar now) throws ParseException {
+        return blackoutTime.canSendAtTime(now, window);
+    }
+
+    private static boolean canSendBetween(long startUtc, long endUtc, long nowUtc) {
```

Comment:
canSendBetween is a misleading function name, because it does not check if it can send between, we know that it cannot send between, it checks if now is between. ... something like "timeNotInBlackoutRange"